app/src/index.html

```diff
@@ -1,11 +1,49 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <title>Decentralized Prediction Market Experiments</title>
     <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600" rel="stylesheet">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-109654521-2');
+    </script>
```

Comment:
This is old Google Analytics tracking from FCLA, let's remove?

Reply:
Removed with: https://github.com/flyingcarpet-network/prediction-market/pull/1/commits/9d246f5d847add6c8215ce9c2842f1d747d184a9